### DISCUSSION TITLE
Blurred Dicord + Firefox zoom to much

### DISCUSSION THREAD

**Discussion by audihop354** [prasanthrangan/hyprdots#993] [Q&A]
Hi Guys, this is the 1st time I try this hyprdots. There are some questions I would like to ask.
1) My Firefox is zoom too much. I dont know how to dicuss this but it kinda big and maybe everyone can see my box message when I chat to my friend. How can I config it.
2) Same to my FF. Discord zoom too much so it become blurred like this pic. How can I fix all these 2 problem thanks yall
![240228_19h59m24s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/111374177/9d8f3770-64fb-4664-ac42-245822442d8c)

**Comment by kRHYME7** [comment]
Have you set your scaling to 1?  

Please read the hyprland wiki. https://wiki.hyprland.org/

``` xwayland:force_zero_scaling = true ```

Try to add this to your ./userprefs.conf

This should look alright as long as you're not on hidpi monitor.

**Reply by audihop354** [reply]
I did it, but I think it's because the monitor when I plug in HDMI (I use a laptop), the HDMI side is fine, but the laptop screen side is still the same.

**Reply by kRHYME7** [reply]
How about adjusting per monitor?
